rust-toolchain

```diff
@@ -1 +1 @@
-1.43.1
+nightly-2020-06-06
```

Comment:
Admission: this choice of nightly is totally arbitrary. Nightly is still needed for `unwind_attributes`, and I wanted to pick a newer one.